packs/spells/hand-of-the-apprentice.json

```diff
@@ -29,7 +29,37 @@
             "value": "500 feet"
         },
         "requirements": "",
-        "rules": [],
+        "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "predicate": [
+                            "spellcasting:cha"
+                        ],
+                        "value": "charisma"
+                    },
+                    {
+                        "predicate": [
+                            "spellcasting:int"
+                        ],
+                        "value": "intelligence"
+                    },
+                    {
+                        "predicate": [
+                            "spellcasting:wis"
+                        ],
+                        "value": "wisdom"
+                    }
+                ],
+                "flag": "handOfApprentice",
+                "key": "ChoiceSet"
+            },
+            {
+                "key": "RollOption",
+                "option": "focus-spell:{item|flags.pf2e.rulesSelections.handOfApprentice}"
+            }
```

Comment:
?

Reply:
This sets the Roll Option as a variable from the previous choice set for predicate purposes. 

The way it works - the choice set provides a value based on `spellcasting:x`. This is then set to the RollOption, which in turn allows the effect to use the correct spellcasting attribute. 

I've been toiling at this for weeks trying to get this to work and this method that worked.